### ISSUE TITLE
No reads are aligned?

### ISSUE THREAD

**Issue by woodoo46** [lbcb-sci/graphmap2#10]
Hi there,

I used this command to align pacbio reads:

graphmap2 -r hs38DH.fa -d test_100reads.fastq -o graph_map2.test.sam

It seems that NO reads were aligned? Or did I miss anything?

Thanks!

**Comment by jmaricb** [comment]
Hi @woodoo46 ,

did the mapping process finished successfully. If not, can you tell me what reads and reference are you using and I will try to align it on my side and let you know what happens. 

Thanks.

**Comment by woodoo46** [comment]
I was using pacbio read, and aligning on hg38.

**Comment by jmaricb** [comment]
Yes I can see that. I was wondering if I could download the dataset from somewhere and try it. Do you have the download link?

**Comment by woodoo46** [comment]
Unfortunately, I do not have any ftp site for public, do you have any ftp site I can deposit my test data?

**Comment by jmaricb** [comment]
Yes I do. Can you share your email with me and I will send you the address and credentials?

**Comment by yjx1217** [comment]
Same problem encountered with the latest released version (v0.6.4). I tried both the prebuilt version and my own compiled version and the results are the same. 

For my case, I am using graphmap2 mapping simulated pacbio reads against yeast reference genome. No error has been triggered but no reads got mapped. The same test works fine for minimap2 and ngmlr, so I suspect this problem is graphmap2 specific. 

Please contact me if you need me to share my testing data.

Best,
Jia-Xing

**Comment by jmaricb** [comment]
Hi, did you try using '-x rnaseq' parameter? Are you mapping RNA or DNA reads? 
If you are mapping RNA reads use '-x rnaseq' parameter and if you are mapping DNA reads you can use the old version of Graphmap which is the same as the new one, there is no difference for DNA reads: https://github.com/isovic/graphmap

**Comment by yjx1217** [comment]
I am aligning DNA reads. OK, I will try the older version of graphmap then.  Thanks!

**Comment by sen1019san** [comment]
I am also in trouble with the problem that no read was aligned. I am using graphmap2(v0.6.3) mapping nanopore simulated reads against ITS/16S database. The result was similar to the Description by @yjx1217. No error happened but no reads got mapped. But minimap2 and last work well with the same read and reference.
Thanks a lot for any help.

**Comment by lpryszcz** [comment]
Same error here - I'm aligning E. coli DNA reads from ONT onto E. coli reference. I've tried building graphmap2 myself and also downloading the latest binary. I've tried compressed and uncompressed FastQ. I've rebuilt the index. The effect is the same...  
minimap2 aligns the same FastQ without any problem.  
Any idea?  

```
> graphmap align -r ../ref/ECOLI.fa -d $d.fastq | samtools sort -o $d.graphmap2.bam; samtools index $d.graphmap2.bam
[12:49:04 BuildIndexes] Loading reference sequences.
[12:49:04 SetupIndex_] Loading index from file: '../ref/ECOLI.fa.gmidx'.
[12:49:05 Index] Memory consumption: [currentRSS = 269 MB, peakRSS = 369 MB]
[12:49:05 Run] Hits will be thresholded at the percentil value (percentil: 99.000000%, frequency: 6).
[12:49:05 Run] Minimizers will be used. Minimizer window length: 5
[12:49:05 Run] Automatically setting the maximum allowed number of regions: max. 500, attempt to reduce after 0
[12:49:05 Run] Reference genome is assumed to be linear.
[12:49:05 Run] Only one alignment will be reported per mapped read.
[12:49:05 ProcessReads] Reads will be loaded in batches of up to 1024 MB in size.
[12:49:15 ProcessReads] Batch of 278853 reads (1024 MiB) loaded in 3.82 sec. (40629008 bases)
[12:49:15 ProcessReads] Memory consumption: [currentRSS = 1376 MB, peakRSS = 1376 MB]
[12:53:13 ProcessReads] [CPU time: 1337.74 sec, RSS: 1442 MB] Read: 278853/278853 (100.00%) [m: 0, u: 278853]                                                      
[12:53:14 ProcessReads] Batch of 28675 reads (110 MiB) loaded in 0.49 sec. (40629008 bases)
[12:53:14 ProcessReads] Memory consumption: [currentRSS = 1442 MB, peakRSS = 1442 MB]
[12:53:37 ProcessReads] [CPU time: 1476.66 sec, RSS: 1445 MB] Read: 28675/28675 (100.00%) [m: 0, u: 28675]
[12:53:37 ProcessReads] Memory consumption: [currentRSS = 1445 MB, peakRSS = 1445 MB]
[12:53:37 ProcessReads] All reads processed in 1476.71 sec (or 24.61 CPU min).
[bam_sort_core] merging from 1 files and 1 in-memory blocks...
```

**Comment by jmaricb** [comment]
Hi @lpryszcz, 

since you aligning DNA reads, please use previous version of the tool: https://github.com/isovic/graphmap
Currently, this one is only working for RNA reads.

**Comment by lpryszcz** [comment]
@jmaricb Could you please add information about that, ideally in the beginning of the README file? It'll likely save hours of struggle to prospective users ;)

**Comment by novigit** [comment]
@jmaricb I had the same issue : trying to align nanopore reads against an assembly. Zero reads were aligned, but no error messages. Running the same command with GraphMap instead of GraphMap2 fixed the issue.

I agree with Ipryszcz, please make it clear in the README that GraphMap2 doesn't work with aligning DNA reads!

**Comment by decrevi** [comment]
Are there any plans to incorporate this option into Graphmap2?? It would be very convenient instead of having two installations of Graphmap.
